langchain-core/src/runnables/config.ts

```diff
@@ -14,3 +14,76 @@ export async function getCallbackMangerForConfig(config?: RunnableConfig) {
     config?.metadata
   );
 }
+
+export function mergeConfigs<CallOptions extends RunnableConfig>(
```

Comment:
We aren't using this type param anywhere are we? Should we be?

Reply:
Whoops, looks like a refactor I made accidentally removed it. Re-added.